snorkel/classification/snorkel_classifier.py

```diff
@@ -371,22 +384,36 @@ def score(self, dataloaders: List["DictDataLoader"]) -> Dict[str, float]:
         self.eval()
 
         metric_score_dict = dict()
+        # By default, evaluate all labels on corresponding task of same name
 
         for dataloader in dataloaders:
             results = self.predict(dataloader, return_preds=True)
-            for task_name in results["golds"].keys():
-                metric_scores = self.scorers[task_name].score(
-                    results["golds"][task_name],
-                    results["preds"][task_name],
-                    results["probs"][task_name],
+
+            # By default, evaluate each label_name on task with corresponding task_name
+            eval_dict = {name: name for name in results["golds"].keys()}
+            # Then, override the current results with manually specified mappings
+            eval_dict.update(remap_labels)
+
+            for label_name, task_name in eval_dict.items():
+                # Use the original gold labels, which include abstains
+                golds = (
```

Comment:
yuck, hard to read

```python
gold_labels = dataloader.dataset.Y_dict[label_name]  # type:ignore
gold_labels = gold_labels.numpy().squeeze()
```

Reply:
I blame `black` for formatting :). fixed!